tools/profiles/develop.json

```diff
@@ -23,6 +23,13 @@
         "cxx": ["--cpp", "--no_rtti", "--no_vla"],
         "ld": []
     },
+    "ARMC6": {
+        "common": ["-c", "--target=arm-arm-none-eabi", "-mthumb", "-g", "-O3", "-Os"],
```

Comment:
That's how I read it. You can take a look here: https://developer.arm.com/docs/100066/latest/optimization/optimizing-for-code-size-or-performance